stdlib/collections/__init__.pyi

```diff
@@ -277,9 +277,15 @@ class Counter(dict[_T, int], Generic[_T]):
     def update(self, __m: Iterable[_T] | Iterable[tuple[_T, int]], **kwargs: int) -> None: ...
     @overload
     def update(self, __m: None = ..., **kwargs: int) -> None: ...
+    def __missing__(self, key: _T) -> int: ...
     def __delitem__(self, elem: object) -> None: ...
-    def __eq__(self, other: object) -> bool: ...
-    def __ne__(self, other: object) -> bool: ...
+    if sys.version_info >= (3, 10):
+        def __eq__(self, other: object) -> bool: ...
+        def __ne__(self, other: object) -> bool: ...
+    else:
+        def __eq__(self, __other: object) -> bool: ...
+        def __ne__(self, __other: object) -> bool: ...
```

Comment:
These don't seem to exist on 3.9, 3.8, 3.7 or 3.6.

Reply:
You're right, it inherits them from `__dict__`.